src/engine/audio/audio.main.ts

```diff
@@ -87,16 +88,22 @@ export const AudioModule = defineModule<IMainThreadContext, MainAudioModule>({
     const audioContext = new AudioContext();
 
     // hack - must play something thru the audio context for media streams to activate
-    (() => {
-      const osc = audioContext.createOscillator();
-      osc.frequency.value = 0;
-      osc.connect(audioContext.destination);
-      osc.start();
-      osc.stop();
-    })();
+    const osc = audioContext.createOscillator();
```

Comment:
This seems unrelated to this PR, it'd generally be better to split this out, but it's ok for now. What is this doing though?

Reply:
it's a limiter with a bit of headroom to protect speakers from spikes in amplitude and take the edge off of hard clipping from the decibel ceiling when audio gets too loud/chaotic